src/main/java/org/folio/dataexp/controllers/MappingProfileController.java

```diff
@@ -113,7 +114,7 @@ public ResponseEntity<Void> putMappingProfile(UUID mappingProfileId, MappingProf
     userInfo.setUserName(user.getUsername());
     mappingProfile.setUserInfo(userInfo);
 
-    var metadata = mappingProfile.getMetadata();
+    var metadata = isNull(mappingProfile.getMetadata()) ? new Metadata() : mappingProfile.getMetadata();
```

Comment:
I don't think it's fully correct, metadata should be created during POST

Reply:
updated

Reply:
still not clear what you updated, we create new mapping profile and for some reasons during update it's metadata is null, but in DB it's saved, please fix root cause and not consequence